benchmarks/benchmark8.ipynb

```diff
@@ -129,20 +129,92 @@
+    "This benchmark is still under construction, but the submission guidelines are included for development purposes"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2019-10-28T18:42:26.097304Z",
+     "start_time": "2019-10-28T18:42:26.083174Z"
+    }
+   },
+   "source": [
+    "## Submission Guidelines\n",
+    "\n",
+    "Please use the [upload form]({{ site.baseurl }}/simulations/upload_form/) to upload your results.\n",
+    "\n",
+    "In addition to that specified, further data to upload can include a Youtube video, images of the nuclei at different times, or the entire phase field variable at different times. These are not required, but will help others view your work.\n",
+    "\n",
+    "### Part (a)\n",
+    "\n",
+    "Run three cases with starting seeds with $r_0 = r^*$, $r_0 = 0.99 r^*$, and $r_0 = 1.01 r^*$. Follow the time evolution of the system for $t = 100$ units. The following data should be collected for upload as frequently as possible.\n",
+    "\n",
+    " - The solid fraction in the domain (the integral $\\int_V \\phi dV / \\int_V dV$)\n",
+    " \n",
+    " - The free energy, $\\mathcal{F}$\n",
+    " \n",
+    "One PFHub upload is required with two data contributions for each of the three simulations (for a total of six data contributions). The data contributions should be named as `solid_fraction_1` and `free_energy_1`, `solid_fraction_2` and `free_energy_2` as well as `solid_fraction_3` and `free_energy_3` where `_1`, `_2` and `_3` refers to $r_0 = r^*$, $r_0 = 0.99 r^*$, and $r_0 = 1.01 r^*$, respectively. The data contributions should be stored in either CSV of JSON format. The columns name for time and quantity need to be identified correctly on the upload form for each data contribution. The data should probably be stored in three separate files for each simulation and each file with three columns for time, free energy and solid fraction.\n",
+    "\n",
+    "Further data to upload can include a Youtube video, images of the nucleus at different times or the entire phase field variable at different times. These are not required, but will help others view your work.\n",
+    "\n",
+    "### Part (b)\n",
+    "\n",
+    "Follow the time evolution of the system for $t = 400$ units. The following data should be collected for upload as frequently as possible.\n",
+    "\n",
+    " - The solid fraction in the domain (the integral $\\int_V \\phi dV / \\int_V dV$)\n",
+    " \n",
+    " - The free energy, $\\mathcal{F}$\n",
+    " \n",
+    "Two data contributions are required and should be named as `solid_fraction` and `free_energy`, and stored in either CSV or JSON format. The results can be stored in two separate files or one single file as long as each column name for time and quantity is identified on the upload form. In addition, the phase field snapshot at time $t=80$ is required. The data should be named as `image` and stored in `PNG` or `TIFF` format.\n",
```

Comment:
Similar to Part (a), simply state what files we're looking for. "Data contributions" is a meaningful abstraction, but is also needlessly confusing.

Reply:
It now reads

> One PFHub upload is required with one data file for each of the three simulations (for a total of three data files). The data sets should be named as solid_fraction_1 and free_energy_1, solid_fraction_2 and free_energy_2 as well as solid_fraction_3 and free_energy_3 where _1, _2 and _3 refers to part (1), (2) and (3), respectively.

The problem is that I can't just refer to data files. We might have three data files, but we still need six data sets that are labeled correctly with the "Short name of data" field on the upload form.